packages/docusaurus/src/webpack/plugins/CleanWebpackPlugin.ts

```diff
@@ -33,16 +33,6 @@ import path from 'path';
 import {sync as delSync} from 'del';
```

Comment:
The changes in this file basically just removes a ton of dead code.

Reply:
for ref, this v2 clean plugin fork was implemented in https://github.com/facebook/docusaurus/pull/1839

Reply:
I'm not 100% sure we need it BTW, Gatsby and Next don't clean /build before building and have some good reasons to do so. But we'll explore that later.